reconcile/utils/oc.py

```diff
@@ -439,6 +440,26 @@ def get_service_account_username(user):
         name = user.split('/')[1]
         return "system:serviceaccount:{}:{}".format(namespace, name)
 
+    def recycle_orphan_pods(self, dry_run, namespace, resource):
```

Comment:
Seeing `validate_pod_ready` function, I don't see where this method of recycling would work apart from StatefulSets so I would name the functions accordingly so they don't give the impression of being more general than they currently are.

Reply:
renamed to `recycle_sts_orphan_pods` and added protection